test/flycheck-test.el

```diff
@@ -3881,6 +3881,33 @@ Why not:
     (flycheck-ert-should-syntax-check
      "language/rust/lib-main/src/main.rs" 'rust-mode)))
 
+(flycheck-ert-def-checker-test rust-cargo rust notes
+  ;; We only get notes on the first build. Ensure we start from a
+  ;; clean directory each time.
+  (call-process-shell-command
```

Comment:
Do we really need a shell?  If not, it would be faster to just do:
```
(call-process "cargo" nil nil nil "clean" "--manifest-path" (expand-file-name ...))
```